oteapi/strategies/transformation/celery_remote.py

```diff
@@ -74,21 +75,16 @@ class CeleryRemoteStrategy:
 
     def get(self, session: "Optional[Dict[str, Any]]" = None) -> SessionUpdateCelery:
         """Run a job, return a job ID."""
-        celery_kwargs = {}
         if session:
             self._use_session(session)
-            celery_kwargs = session.copy()
```

Comment:
We need to use the valid fields of the session as the configuration. I.e. the specific configuration might come from the conf-object directly OR through the session.